### PR TITLE
[react-google-recaptcha] Add onErrored prop type

### PR DESCRIPTION
**Pull request by MaxwellBo**
- Added the `onErrored?: () => void;` prop, with description sourced from https://developers.google.com/recaptcha/docs/display

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/recaptcha/docs/display and https://github.com/dozoisch/react-google-recaptcha/blob/9aa2218e7b4d633f329b8ea7b708afb880a06f80/src/recaptcha.js#L143
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-google-recaptcha/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for React Google Recaptcha 0.10
+// Type definitions for React Google Recaptcha 0.11
```
Comment (uniqueiniquity) [inline types/react-google-recaptcha/index.d.ts]: From a quick look, it seems like `onErrored` is not used as part of props in `react-google-recaptcha@0.11`. Can you change the version to reflect the version of `react-google-recaptcha` that you want to describe? Thanks!

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/react-google-recaptcha/index.d.ts (hunk 1 above)
Comment (MaxwellBo) [reply types/react-google-recaptcha/index.d.ts]: It's been bumped to v1.0, to coincide with when it was added https://github.com/dozoisch/react-google-recaptcha/commit/2d5b964b801cb54858830abeb45c645c6e83818b

Comment (uniqueiniquity) [reply types/react-google-recaptcha/index.d.ts]: @MaxwellBo Thanks for doing that! Unfortunately, given that it was introduced in a new major version, and it seems like v0 is still used, there's another thing I'll need you to do. Can you follow the steps found [here](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version)? Thanks again!

**Comment by MaxwellBo** [comment]
Can I get a followup on this @uniqueiniquity ?

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]
Waiting to merge until major version bump process is followed, as mentioned before. Directions for this can be found [here](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version).

**Comment by MaxwellBo** [comment]
Hi @uniqueiniquity, sorry for the delay. I was having issues performing the major version bump because all the links to the procedure are dead

**Review by uniqueiniquity — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Sorry about that, looks like some things in the README got renamed. Thanks for making the changes!

Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**